tutorials-v4/qobjevo.md

```diff
@@ -17,14 +17,15 @@ Made by Eric Giguere, updated by Jake Lishman
 
 ```python
 # Basic setup
-import qutip
 import numpy as np
+import qutip
+
 size = 4
 t = 1.0
 a = qutip.destroy(size)
 ad = qutip.create(size)
 n = qutip.num(size)
-I = qutip.qeye(size)
+iden = qutip.qeye(size)
```

Comment:
I'm not a big fan of `iden` since we mostly use capitals for H (although `a`, `ad` and `n`, especially `n` fail to follow that pattern). Maybe `II` or `Id`?